openfaas/post_install.md

```diff
@@ -4,20 +4,38 @@ Thank you for deploying [OpenFaaS](https://github.com/openfaas/faas) to Civo's k
 
 ## Obtain access
 
-The OpenFaaS gateway has been made available through a `LoadBalancer` on port 80 on each node through the use of a Host Port. This is a feature of `k3s`.
+The OpenFaaS gateway has been made available through a `NodePort` on port `31112` on each node.
 
 * Go to your Civo dashboard, click Kubernetes and then your OpenFaaS Cluster.
 
 Look for the DNS entry that you find there, it may look something like `6c1c1646-25cf-44f0-9bd5-53ee35cd7c84.k8s.civo.com`
 
+This DNS record which points at each of the nodes in your cluster.
+
 * Set the following URL:
 
 ```sh
 export DNS="" # As per dashboard
 export OPENFAAS_URL=http://$DNS:31112
 ```
 
-* Download your `kubeconfig` file from the Civo dashboard.
+### Get your kubeconfig
+
+Pick A or B:
+
+* A) Get your kubeconfig via command-line
+
+```
+civo k8s ls
+
+civo k8s kubeconfig --save <CLUSTER_NAME>
+
+kubectl config set-context <CLUSTER_NAME>
+```
+
+* B) Get your kubeconfig via the Dashboard
+
+* Either download your `kubeconfig` file from the Civo dashboard.
```

Comment:
This "Either" isn't right when it's the second option and the first was already given?

Reply:
Good point